README.md

```diff
@@ -85,9 +85,7 @@ This may be acceptable for same-origin transitions, since the application could
 
 ## Same-Origin vs. Cross-Origin Navigation
 
-For navigation scenarios (scenario #1), it may be relevant whether the target URL has the same origin as the existing page. Specifically, since the user has already interacted with the origin, it may be reasonable for implementations to be more permissive in such cases.
-
-However, allowing one but not the other would lead to an [Inconsistent User Experience](https://github.com/immersive-web/webxr/blob/master/designdocs/navigation.md#inconsistent-user-experience).
+For navigation scenarios (scenario #1), it may be relevant whether the target URL has the same origin as the existing page. Specifically, since the user has already interacted with the origin, it may be reasonable for implementations to be more permissive in such cases. UAs could navigate seamlessly on the same origin and show a modal dialog on cross-origin. Users would have the change to evaluate the URL and decide to continue or return to previous origin.
```

Comment:
"show a modal dialog on cross-origin navigation.  Users would have the chance" ... "return to the previous"